src/main/java/org/ibp/api/rest/inventory_new/TransactionResource.java

```diff
@@ -99,4 +107,19 @@ public long getCount() {
 
 	}
 
+	@ApiOperation(value = "Create Transaction", notes = "Create a new transaction")
+	@RequestMapping(value = "/crops/{cropName}/lots/{lotId}/transactions", method = RequestMethod.POST)
+	@ResponseBody
+	public ResponseEntity<Integer> createTransaction(
+		@PathVariable final String cropName,
+		@PathVariable final String lotId,
+		@ApiParam("Transaction to be created")
+		@RequestBody final TransactionDto transactionDto) {
+
+
+		final String userName = SecurityUtil.getLoggedInUserName();
```

Comment:
@vmaletta please do this:
`final WorkbenchUser loggedInUser = this.securityService.getCurrentlyLoggedInUser();`